docs/features/software-catalog/system-model.md

```diff
@@ -73,6 +73,8 @@ these entities using the following (optional) concepts:
   function
 - **Domains** relate entities and systems to part of the business
 
+![](../../assets/software-catalog/software-model-entities.drawio.svg)
```

Comment:
You don't have to, but you can make this architecture diagram zoomable on backstage.io/docs, like Medium. 😁 

```
<img data-zoomable src="../../assets/software-catalog/software-model-entities.drawio.svg" />
```

Ref: https://github.com/backstage/backstage/pull/3488

Reply:
Cool feature, good to know. I checked out how the pictures look on smaller devices, not sure if this is necessary.

Reply:
Yep, sounds good! It was mostly a FYI and plug, anyway :)